### ISSUE TITLE
Opportunity to turn of the sun

### ISSUE THREAD

**Issue by Granada1** [TheGameCreators/GameGuruRepo#2101]
Can we have the choice to turn of the sun for completely internal levels guys

**Comment by ArgentArts** [comment]
Agreed. It's been asked for a few times. At least a simple, one-click way to turn off the sky/sun instead of having to adjust dozens of controls to try to achieve this.

**Comment by synchromesh62** [comment]
Being more interested in indoor work for a more unique game creation experience i have to agree.
It just effects the environment to much and its almost impossible to hide.
